### PR TITLE
[editor] Mapsme 1655 incorrect floor count validation

### PR DESCRIPTION
**Pull request by milchakov**
https://jira.mail.ru/browse/MAPSME-1655
@ygorshenin 
@mpimenov

### CODE REVIEW

File: indexer/editable_map_object.cpp [hunk 1]
```diff
@@ -202,6 +202,9 @@ bool EditableMapObject::ValidateBuildingLevels(string const & buildingLevels)
   if (buildingLevels.size() > 18 /* max number of digits in uint_64 */)
     return false;
 
+  if ('0' == buildingLevels.front())
```
Comment (ygorshenin) [inline indexer/editable_map_object.cpp]: tests? :)

Comment (milchakov) [inline indexer/editable_map_object.cpp]: oops

**Comment by mpimenov** [comment]
LGTM

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**